tables/hbo.tbl

```diff
@@ -0,0 +1,275 @@
+letter ֒ 46-234  # segol - isolated
+letter ֓ 46-146  # shalshelet
+letter ֔ 2  # zaqef qatan
+letter ֕ 46-1245  # zaqef gadol
+letter ֖ 46-2345  # tipeha
+letter ֗ 46-1235  # revia
+letter ֘ 46-145  # zarqa
+letter ֙ 46-34  # pashta
+letter ֚ 46-245  # yetiv
+letter ֛ 46-1236  # tevir
+letter ֜ 46-1  # geresh
+letter ֝ 46-16  # geresh muqdam
+letter ֞ 46-12  # gershayim
+letter ֟ 46-1345  # qarney para
+letter ֠ 46-2356  # telisha gedola
+letter ֡ 46-236  # pazer
+letter ֢ 46-23  # atnah hafukh
+letter ֣ 46-136  # munah
+letter ֤ 46-125  # mahapakh
+letter ֥ 46-134  # merkha
+letter ֦ 46-13  # merkha kefula
+letter ֧ 46-145  # darga
+letter ֨ 46-1234  # qadma
+letter ֩ 46-12345  # telisha qetana
+letter ֪ 46-13456  # yerah ben yomo
+letter ֫ 46-135  # ole
+letter ֬ 46-24  # iluy
+letter ֭ 46-24  # dehi
+letter ֮ 46-1356  # zinor
+letter ֯ 456-134  # masora circle
+letter ְ 3  # sheva
+letter ֱ 26  # hataf segol
+letter ֲ 25  # hataf patah
+letter ֳ 345  # hataf qamats
+letter ִ 24  # hiriq
+letter ֵ 34  # tsere
+letter ֶ 15  # segol
+letter ַ 14  # patah
+letter ָ 126  # qamats
+letter ֹ 456-246  # holam Malay
+letter ֺ 135  # holam haser
+letter ֻ 346  # qubuts
+letter ּ 5  # dagesh
+letter ּו 136  # shuruq
+letter ּוְ 136-3  # shuruq sheva
+letter ֽ 4  # meteg
+letter ֿ 45  # rafe
+letter ׁ 456-4  # shin dot
+letter ׂ 456-45  # sin dot
+letter ׄ 456-5  # upper dot
+letter ׅ 456-56  # lower dot
+letter ׆ 456-1345  # nun hafukha
+letter ׇ 126  # qamats qatan
+letter א 1  # alef
+letter ב 1236  # bet
+letter בְּ 12-3  # bet with dagesh sheva
+letter בּ 12  # bet with dagesh
+letter ג 1245  # gimel
+letter ד 145  # dalet
+letter דּ 5-145  # dalet with dagesh
+letter דְּ 5-145-3  # dalet with dagesh sheva
+letter ה 125  # hey
+letter הּ 125  # hey - isolated
+letter ו 2456  # vav
+letter וּ 136  # shuruq
+letter ז 1356  # zayin
+letter ח 1346  # het
+letter ט 2345  # tet
+letter י 245  # yod
+letter יּ 5-245  # yod with dagesh
+letter יְּ 5-245-3  # yod with dagesh
+letter ך 16  # final kaf
+letter כ 16  # kaf
+letter כּ 13  # kaf with dadgesh
+letter כְּ 13-3  # kaf with dadgesh sheva
+letter ל 123  # lamed
+letter לּ 5-123  # lamed with dagesh
+letter לְּ 5-123-3  # lamed with dagesh sheva
+letter ם 134  # final mem
+letter מ 134  # mem
+letter מּ 5-134  # mem with dagesh
+letter מְּ 5-134-3  # mem with dagesh sheva
+letter ן 1345  # final nun
+letter נ 1345  # nun
+letter נּ 5-1345  # nun with dagesh
+letter נְּ 5-1345-3  # nun with  dagesh sheva
+letter ס 234  # samekh
+letter ע 1246  # ayin
+letter ף 124  # final pe
+letter פ 1234  # Pe
+letter פּ 5-1234  # pe with dagesh
+letter פְּ 5-1234-3  # pe with dagesh sheva
+letter ץ 2346  # final tsadi
+letter צ 2346  # tsadi
+letter ק 12345  # qof
+letter קּ 5-12345  # qof with dagesh
+letter קְּ 5-12345-3  # qof with dagesh sv
+letter ר 1235  # resh
+letter שְׁ 146-3  #  Shin with dot and sheva
+letter שְׂ 156-3  #  Sin sheva
+letter שׁ 146  # shin with dot
+letter שׂ 156  # sin
```

Comment:
Is it valid to only have a ש, so without the dot? If so, I think that deserves a rule by itself.

Reply:
Plus one to this comment - shin without a dot should totally get its own line. In Yiddish, the the unmarked version i always shin, dots 146, so my suggestion is to go with that, although biblical Hebrew is not me area of expertise.